packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -393,7 +402,8 @@ export class MgtTodo extends MgtTasksBase {
     const taskCheckContent = html`${getSvg(SvgIcon.CheckMark)}`;
 
     return html`
-      <fluent-checkbox id=${task.id} class=${taskClasses} checked @click="${() => this.handleTaskCheckClick(task)}">
+      <fluent-checkbox id=${task.id} class=${taskClasses} checked ?disabled=${this.readOnly} @click="${() =>
+        this.handleTaskCheckClick(task)}">
```

Comment:
formatting, if we get to the point where it wants to wrap then we should stack the attributes:
```suggestion
      <fluent-checkbox
        id=${task.id}
        class=${taskClasses}
        checked
        ?disabled=${this.readOnly}
        @click="${() => this.handleTaskCheckClick(task)}"
      >
```